handbook/product/design/reviewing_design/index.md

```diff
@@ -13,3 +13,11 @@ As a design team, it’s our responsibility [to communicate expectations clearly
 As a teammate outside the design team, it’s your responsibility to understand [when and how your feedback will be most valuable](../../../communication/seeking-and-giving-feedback.md#giving-feedback).
 
 We have a [set of guidelines for how to seek and give great feedback](../../../communication/seeking-and-giving-feedback.md).
+
+## Reviews in Figma
+
+To make reviewing designs easier, we use the following tools in Figma:
+
+- [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **design**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).
+- [**Helper cards:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=2814%3A6218) to annotate the design and highlight important details. Types of cards: annotation (to provide additional context), todo, feedback requested (to give more context regardign what feedback designer is looking for) and metric collection (see [defining metrics](https://about.sourcegraph.com/handbook/product/design/metrics/defining-metrics)).
+- [**Figma comments:**](https://help.figma.com/hc/en-us/articles/360041068574-Add-comments-to-files) for ongoing discussions with reviewers. Please note: to review longer copy, use Google Doc instead of Figma comments.
```

Comment:
```suggestion
- [**Figma comments:**](https://help.figma.com/hc/en-us/articles/360041068574-Add-comments-to-files) for ongoing discussions with reviewers. You may find it easier to review and collaborate on copy longer than 1-2 lines (such as a complete modal) using a Google Doc instead of Figma comments.
```

Suggestion based on @jjinnii's feedback. I also suggest changing it to "You may find it easier," because it's not always the case, but rather more about making sure we know it's an option that can help.